client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -46,27 +48,38 @@ export function requestMedia( { dispatch, getState }, { siteId, query } ) {
 		.catch( () => dispatch( failMediaRequest( siteId, query ) ) );
 }
 
-export function requestMediaItem( { dispatch, getState }, { siteId, mediaId } ) {
-	if ( isRequestingMediaItem( getState(), siteId, mediaId ) ) {
```

Comment:
I did a quick usage search. I believe we can remove the `isRequestingMediaItem` along with the `mediaItemRequests` reducers in another PR.